synapse_spamcheck_badlist/bad_list_filter.py

```diff
@@ -101,6 +113,11 @@ async def can_we_check_md5(self) -> bool:
         """
             Check whether the MD5 database exists, caching the result.
```

Comment:
I don't think its worth having this check TBH, when this is deployed we should absolutely have the table, and if we don't we probably want to start erroring anyway.